### PR TITLE
Fix dml stream leakage in Proxy

### PR DESCRIPTION
**Pull request by longjiquan**
Signed-off-by: longjiquan <jiquan.long@zilliz.com>
issue: #19438

### CODE REVIEW

**Comment by longjiquan** [comment]
rerun ut

**Comment by longjiquan** [comment]
rerun u

**Comment by longjiquan** [comment]
rerun ut

**Comment by longjiquan** [comment]
rerun ut

**Comment by longjiquan** [comment]
rerun ut

File: internal/rootcoord/expire_cache.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	// if only collNames are specified, invalidate the collection meta cache with the specified collectionName
+	for _, collName := range collNames {
+		req := proxypb.InvalidateCollMetaCacheRequest{
+			Base: &commonpb.MsgBase{
+				MsgType:   0, //TODO, msg type
+				MsgID:     0, //TODO, msg id
+				Timestamp: ts,
+				SourceID:  c.session.ServerID,
+			},
+			CollectionName: collName,
```
Comment (czs007) [inline internal/rootcoord/expire_cache.go:60]: better to also assign the collectionID in the InvalidateCollMetaCacheRequest.

**Pull request merged**